src/corehost/packaging/projects/Microsoft.NETCore.App/project.json

```diff
@@ -0,0 +1,45 @@
+{
+  "dependencies": {
+    "Microsoft.CodeAnalysis.CSharp": "1.3.0-beta1-20160321-04",
+    "Microsoft.CodeAnalysis.VisualBasic": "1.3.0-beta1-20160321-04",
+    "Microsoft.CSharp": "4.0.1-rc2-23931",
+    "Microsoft.VisualBasic": "10.0.1-rc2-23931",
+    "NETStandard.Library": "1.5.0-rc2-23931",
+    "System.Buffers": "4.0.0-rc2-23931",
+    "System.Collections.Immutable": "1.2.0-rc2-23931",
+    "System.ComponentModel": "4.0.1-rc2-23931",
+    "System.ComponentModel.Annotations": "4.1.0-rc2-23931",
+    "System.Diagnostics.DiagnosticSource": "4.0.0-rc2-23931",
+    "System.Diagnostics.Process": "4.1.0-rc2-23931",
+    "System.Dynamic.Runtime": "4.0.11-rc2-23931",
+    "System.Globalization.Extensions": "4.0.1-rc2-23931",
+    "System.IO.FileSystem.Watcher": "4.0.0-rc2-23931",
+    "System.IO.MemoryMappedFiles": "4.0.0-rc2-23931",
+    "System.IO.UnmanagedMemoryStream": "4.0.1-rc2-23931",
+    "System.Linq.Expressions": "4.0.11-rc2-23931",
+    "System.Linq.Parallel": "4.0.1-rc2-23931",
+    "System.Linq.Queryable": "4.0.1-rc2-23931",
+    "System.Net.NameResolution": "4.0.0-rc2-23931",
+    "System.Net.Requests": "4.0.11-rc2-23931",
+    "System.Net.Security": "4.0.0-rc2-23931",
+    "System.Net.WebHeaderCollection": "4.0.1-rc2-23931",
+    "System.Numerics.Vectors": "4.1.1-rc2-23931",
+    "System.Reflection.DispatchProxy": "4.0.1-rc2-23931",
+    "System.Reflection.Metadata": "1.3.0-rc2-23931",
+    "System.Reflection.TypeExtensions": "4.1.0-rc2-23931",
+    "System.Resources.Reader": "4.0.0-rc2-23931",
+    "System.Runtime.Loader": "4.0.0-rc2-23931",
+    "System.Security.Cryptography.Algorithms": "4.1.0-rc2-23931",
+    "System.Security.Cryptography.Encoding": "4.0.0-rc2-23931",
+    "System.Security.Cryptography.Primitives": "4.0.0-rc2-23931",
+    "System.Security.Cryptography.X509Certificates": "4.1.0-rc2-23931",
+    "System.Threading.Tasks.Dataflow": "4.6.0-rc2-23931",
+    "System.Threading.Tasks.Extensions": "4.0.0-rc2-23931",
+    "System.Threading.Tasks.Parallel": "4.0.1-rc2-23931",
+    "System.Threading.Thread": "4.0.0-rc2-23931",
+    "System.Threading.ThreadPool": "4.0.10-rc2-23931"
+  },
+  "frameworks": {
+    "netstandard1.5": { }
+  }
```

Comment:
See https://github.com/dotnet/cli/pull/2274/files#diff-a4a50058597e5e9e728f0ac284a74259R11 which is a ProjectReference to the host package project, so it is always the live built version.